### PR TITLE
Set ID tasks can be delegated to local host.

### PR DESCRIPTION
**Pull request by brett060102**
The tasks that set the test id can be delegated to localhost.

### CODE REVIEW

File: runner/ansible/roles/1.1.1/tasks/main.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@
   set_fact:
     test_id: '1.1.1'
     key_name: 'token'
+  delegate_to: localhost
+
```
Comment (stefanotorresi) [inline runner/ansible/roles/1.1.1/tasks/main.yml:8]: nit: one more line break than needed 🙂

**Review by stefanotorresi — APPROVED** [review]
LGTM, other than a mini nitpick

File: runner/ansible/roles/1.1.1/tasks/main.yml (hunk 1 above)
Comment (brett060102) [reply runner/ansible/roles/1.1.1/tasks/main.yml:8]: removed

**Comment by arbulu89** [comment]
@brett060102 Closing this as these tasks are removed in https://github.com/trento-project/trento/pull/257

**Pull request closed without merging**